fastchat/llm_judge/gen_api_answer.py

```diff
@@ -29,6 +29,8 @@ def get_answer(
 ):
```

Comment:
let's add an assertion that we can't have both `args.force_temperature` and `required_temperature`